bazel-java-deps.bzl

```diff
@@ -103,7 +103,7 @@ def install_java_deps():
             "org.playframework.anorm:anorm_{}:2.6.8".format(scala_major_version),
             "org.playframework.anorm:anorm-akka_{}:2.6.8".format(scala_major_version),
             "com.typesafe.scala-logging:scala-logging_{}:3.9.2".format(scala_major_version),
-            "com.zaxxer:HikariCP:3.2.0",
+            "com.zaxxer:HikariCP:5.0.1",
```

Comment:
Interesting that it works upgrading HikariCP. I remember we were actually forced to not update it on the last attempt because of the conflicting transient dependency of slf4j-api. Did you run `evicted` after these updates on the test branch?